homeassistant/components/songpal/strings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "flow_title": "Sony Songpal {name} ({host})",
+    "step": {
+      "user": {
+        "data": {
+          "name": "Name (Optional)",
```

Comment:
I don't think you need to specify that this is an optional parameter. The UI should handle this for you